### ISSUE TITLE
how to reset mongodb migration?

### ISSUE THREAD

**Issue by code-K** [ParsePlatform/parse-server#364]
Hi there i already started a mongodb migration a few days before and now switched/re-created a new server and wanted to start the migration process again.

and now i'm getting the following errors when clicking on "migrate" button:

CURRENT PROGRESS (*****)

![bildschirmfoto 2016-02-11 um 21 59 47](https://cloud.githubusercontent.com/assets/1224750/12990473/d7d915a8-d10a-11e5-80bf-cdeacb74220e.png)

what can i do now to reset the earlier migration and start from scratch again??

Thanks heaps and best regards,
Christian

**Comment by gfosco** [comment]
You should be able to clear out the mongo database and start a new migration from the settings page.

**Comment by emkman** [comment]
@gfosco I initiated a migration on my production app to MongoLab and the data imported fine, but when I hit Finalize it said it failed to commit. However, I have confirmed that new writes are indeed going to the external MongoLab instance! If I go back in settings it will let me enter a new connection string. Should I just leave it alone? Do I have to worry that it will reset or revert or stop writing to the external instance?
